math/OpenBLAS/Portfile

```diff
@@ -46,19 +72,27 @@ if {[string first "-devel" $subport] > 0} {
 
     github.livecheck.branch develop
 
+    #Not using precompiled binaries in -devel support
+    default_variants-append +native
+
 } else {
 
     github.setup    xianyi OpenBLAS 0.3.6 v
     checksums       rmd160 1e64aaaf26c92e8d49e3a9a61b1c43c7b99ad647 \
                     sha256 57e9fab61e1993005ab9f5d6b99929febbeddee3fda6d0e446b515ccff4f4fe4 \
                     size   11923774
-    revision        0
+    revision        1
 
     conflicts       OpenBLAS-devel
 
     patchfiles      patch-libnoarch.release.diff \
                     patch-linkLib.release.diff  \
                     patch-OpenBLAS-i386-Apple.diff
+
+    notes "
+        This version is built based on a base architecture for convenience, 
+        which may not be optimized for your system. To build a version 
+        customized for your machine, use the +native variant"
```

Comment:
Are these notes skipped if the native variant is selected ? They should be, and I don’t really see just looking at the code how that would happen.

Reply:
Fixed.